engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowHeartbeatConfig.scala

```diff
@@ -0,0 +1,64 @@
+package cromwell.engine.workflow.workflowstore
+
+import java.util.UUID
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.circe.generic.auto._
+import io.circe.literal._
+import io.circe.syntax._
+import io.circe._
+import net.ceedubs.ficus.Ficus._
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+
+/**
+  *
+  * @param cromwellId         Identifier for this Cromwell for horizontaling purposes.
+  * @param ttl                Entries in the workflow store with heartbeats older than `ttl` ago are presumed abandoned and up for grabs.
+  * @param heartbeatsPerTtl   How many heartbeats each `WorkflowActor` sends within an interval of duration `ttl` and how frequently
```

Comment:
Why not just define this as heartbeatInterval?  It looks like everywhere it is used, it's in that form after doing the math.  Also might eliminate problems where someone extends the TTL, expecting the heartbeat to be the same rate and not realizing that they have do up that as well.

Reply:
I thought it was more intuitive to think of it as the number of heartbeats per TTL but it could certainly be done as an interval instead. You're certainly right it's an interval everywhere internally. If this does go to an interval I'd want to keep similar sanity-bounding as exists now (the heartbeat interval could not be longer than say 1/3 of the TTL).